packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -588,6 +588,40 @@ export default class TabAdvanced extends Component {
     blockedOperations => STOP_OPERATIONS.every(op => op in blockedOperations)
   )
 
+  _getIsMigrationAllowed = createSelector(
+    () => this.props.vm?.blockedOperations,
+    blockedOperations =>
+      blockedOperations !== undefined && !['migrate_send', 'pool_migrate'].every(op => op in blockedOperations)
+  )
+
+  _onChangeAllowMigration = allow => {
+    const blockedOperations = this.props.vm.blockedOperations
+    const reasons = JSON.stringify(this.props.vm.blockedOperations)
+
+    const toggleBlockedOperations = () =>
+      editVm(this.props.vm, {
+        blockedOperations: Object.assign.apply(
+          null,
+          ['migrate_send', 'pool_migrate'].map(op => ({ [op]: allow ? null : 'Migration blocked' }))
```

Comment:
This reason does not provide any information, you can remove it:

```suggestion
          ['migrate_send', 'pool_migrate'].map(op => ({ [op]: allow ? null : true }))
```

Reply:
Done